Configure

```diff
@@ -1012,9 +1012,13 @@ if (scalar(@seed_sources) == 0) {
     print "Using os-specific seed configuration\n";
     push @seed_sources, 'os';
 }
-die "Cannot seed with none and anything else"
-    if scalar(grep { $_ eq 'none' } @seed_sources) > 0
-        && scalar(@seed_sources) > 1;
+if (scalar(grep { $_ eq 'none' } @seed_sources) > 0) {
+    die "Cannot seed with none and anything else" if scalar(@seed_sources) > 1;
+    warn <<_____ if scalar(@seed_sources) == 1;
+You have selected NO seed source.  Some functions (such as creating keys)
+will not work.
```

Comment:
Well they won't work without manual seeding. Perhaps a pointer towards the RAND_DRBG(7) page?

Reply:
My humble suggestion:

``` 
You have selected the --with-rand-seed=none option, which effectively disables
automatic reseeding of the OpenSSL random generator. All operations depending on
the random generator such as creating keys will not work unless the random generator
is seeded manually by the application.

Please read the 'Note on random number generation' section in the INSTALL instructions 
document and the RAND_DRBG(7) manual page for more details.
```

Reply:
correction: instructions ~~document~~